doc/stdenv/stdenv.chapter.md

```diff
@@ -309,13 +309,51 @@ The attribute can also contain a list, a script followed by arguments to be pass
 passthru.updateScript = [ ../../update.sh pname "--requested-release=unstable" ];
 ```
 
-The script will be run with the `UPDATE_NIX_NAME`, `UPDATE_NIX_PNAME`, `UPDATE_NIX_OLD_VERSION` and `UPDATE_NIX_ATTR_PATH` environment variables set respectively to the name, pname, old version and attribute path of the package it is supposed to update.
+Finally, the attribute can be an attribute set, which, in addition to the command, can optionally provide the canonical attribute path to use for the list of extra features the script supports.
+
+```nix
+passthru.updateScript = {
+  command = [ ../../update.sh pname ];
+  attrPath = pname;
+  supportedFeatures = [ … ];
+};
+```
+
+##### How update scripts are executed? {#var-passthru-updateScript-execution}
+
+For information about how to run update scripts, run `nix-shell maintainers/scripts/update.nix` in the root of Nixpkgs repository. The script offers several modes for finding and executing update scripts.
+
+Each script will be run with the `UPDATE_NIX_NAME`, `UPDATE_NIX_PNAME`, `UPDATE_NIX_OLD_VERSION` and `UPDATE_NIX_ATTR_PATH` environment variables set respectively to the name, pname, old version and attribute path of the package it is supposed to update.
```

Comment:
This would be much easier to read as a list with annotations. Please fill in something reasonable, I just made things up to show the idea.

```suggestion
Each script will be run with the following environment variables set according to the package being updated:

 - `UPDATE_NIX_NAME` - ???
 - `UPDATE_NIX_PNAME` - package name
 
   Example: `clang`
   
-  `UPDATE_NIX_OLD_VERSION` - old package version to update

   Example: `11.1.0`
   
- `UPDATE_NIX_ATTR_PATH` - attribute path of the package

   Example: `llvmPackages.clang-unwrapped`
   
```

The term "attribute path" apparently is nowhere really defined except implicitly in [`attrsets.nix`](https://github.com/NixOS/nixpkgs/blob/5bc1b01a403e071500a3b0659364fb6ee20391e1/lib/attrsets.nix#L15-L30), one currently has to know that it means roughly "a sequence of attribute names to resolve to get to that value". In this context it's also missing that the attribute this path belongs to, which is probably the one in `all-packages.nix`, but how would one know exactly?